pkg/controller/controller_binding.go

```diff
@@ -45,19 +47,19 @@ func (c *controller) bindingAdd(obj interface{}) {
 	c.bindingQueue.Add(key)
 }
 
-func (c *controller) reconcileServiceInstanceCredentialKey(key string) error {
+func (c *controller) reconcileServiceInstanceCredentialKey(key string) (bool, error) {
```

Comment:
Add godoc comment for this method, make sure you say what the bool argument represents